electronics_model/test_kicad_import.py

```diff
@@ -0,0 +1,90 @@
+import os
+import unittest
+
+import edgir
+from edg import BoardTop
+from electronics_model import Passive
+from electronics_model.KiCadSchematicBlock import KiCadSchematicBlock
+
+
+class KiCadBlock(KiCadSchematicBlock):
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.PORT_A = self.Port(Passive())
+
+    def contents(self) -> None:
+        super().contents()
+
+        self.import_kicad(os.path.join(os.path.dirname(__file__), "resources", "test_kicad_import.net"))
+
+class BlinkyExample(BoardTop):
+    def __init__(self):
+        super().__init__()
+
+        self.kicad = self.Block(KiCadBlock())
+        print("path = " + os.path.dirname(__file__))
```

Comment:
Generally discouraged to have prints in unit tests - unit tests should be fully automated